README.md

```diff
@@ -1,10 +1,17 @@
 # how to open fetch-project-experiment
+
```

Comment:
In the root of `fetch-project-experimentation` 

_specify the name of the project rather than, this project everywhere_

Reply:
ok, I'll fix this..